pkg/api/validation/validation.go

```diff
@@ -3517,3 +3530,17 @@ func isValidHostnamesMap(serializedPodHostNames string) bool {
 	}
 	return true
 }
+
+func sysctlIntersection(a []api.Sysctl, b []api.Sysctl) []api.Sysctl {
+	lookup := make(map[string]struct{}, len(a))
```

Comment:
Then we have to extract the names first. Gives us two loops as well. Or
another helper. Will take a look.

On Wed, Aug 24, 2016 at 5:07 PM, Andy Goldstein <notifications@github.com>
wrote:

> In pkg/api/validation/validation.go
> <https://github.com/kubernetes/kubernetes/pull/27180#discussion_r76074722>
> :
>
> > @@ -3517,3 +3530,17 @@ func isValidHostnamesMap(serializedPodHostNames string) bool {
> >  	}
> >  	return true
> >  }
> > +
> > +func sysctlIntersection(a []api.Sysctl, b []api.Sysctl) []api.Sysctl {
> > +	lookup := make(map[string]struct{}, len(a))
>
> Consider using pkg/util/sets.String and its various helper methods?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27180/files/ac007642cb78b954bdfb27cdac0da7634cb02b04..00d7068d939c4fbc40da44209908e0832792dcee#r76074722>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAskCzRongxKx22ismihaO2MecRYATitks5qjF4ogaJpZM4Iy3_n>
> .
>